src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = reader.readLine();
+            while (value != null) {
+                if (value.startsWith("supply,")) {
+                    String supplyResult = value.substring(value.indexOf(',') + 1);
+                    supply += Integer.valueOf(supplyResult);
+                } else {
+                    String buyResult = value.substring(value.indexOf(',') + 1);
+                    buy += Integer.valueOf(buyResult);
+                }
+                value = reader.readLine();
+            }
```

Comment:
Consider using the split() method to parse the input line instead of using substring() and indexOf(), which could make the code cleaner and easier to understand.